libp2p/transports/quictransport.nim

```diff
@@ -192,19 +187,30 @@ method handle*(m: QuicMuxer): Future[void] {.async: (raises: []).} =
     trace "finished handling stream"
     doAssert(chann.closed, "connection not closed by handler!")
 
-  try:
-    while not m.session.atEof:
-      let stream = await m.session.getStream(Direction.In)
-      asyncSpawn handleStream(stream)
-  except QuicTransportError as exc:
-    trace "Exception in quic handler", msg = exc.msg
+  while not m.session.atEof:
+    let stream =
+      try:
+        await m.session.getStream(Direction.In)
+      except CancelledError:
+        # keep handling, until connection is closed
+        continue
+      except QuicError as exc:
+        if exc.msg == "connection closed":
+          # stop handling, connection was closed
+          break
+        else:
+          # keep handling, until connection is closed. 
+          # this stream failed but we need to keep handling for other streams.
+          trace "QuicMuxer.handler got error while opening stream", msg = exc.msg
+          continue
+    asyncSpawn handleStream(stream)
```

Comment:
`QuicMuxer.handle` must continue until connection is closed. Errors getting single stream should not affect other streams.